.github/workflows/stale.yml

```diff
@@ -0,0 +1,25 @@
+# This workflow closes issues that have been labeled as 'waiting for response' and had no activity for 30 days.
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+name: Close stale 'waiting for response' issues
+
+on:
+  schedule:
+  - cron: '0 0 * * *'
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+
+    steps:
+    - uses: actions/stale@v5
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        close-issue-message: 'This issue was closed because it has been stalled for 30 days with no activity. If the issue still persists, please reopen with the information requested. Thanks.'
+        days-before-issue-close: 30
+        only-labels: 'waiting for response'
```

Comment:
Does this mean that `waiting for response` is the only label that can be on the issue, or does it mean that it will close it as long as it has `waiting for response` as a label?

I would prefer the latter if possible.

Reply:
@AdamYoblick , thanks, I've changed it to `any-of-labels`